test/ts-scenario/config/handlers/sample-query-handler.ts

```diff
@@ -56,13 +48,9 @@ class SampleQueryHandler implements QueryHandler {
  * @param {Network} network The network where transactions are to be evaluated.
  * @returns {QueryHandler} A query handler implementation.
  */
-const createQueryHandler: QueryHandlerFactory = (network: Network, options: any): SampleQueryHandler => {
-	let timeout: number = 3000; // default 3 seconds
-	if (Number.isInteger(options.timeout)) {
-		timeout = options.timeout * 1000; // convert to ms;
-	}
+const createQueryHandler: QueryHandlerFactory = (network: Network): SampleQueryHandler => {
 	const peers = network.channel.getEndorsers(network.mspid);
-	return new SampleQueryHandler(peers, timeout);
+	return new SampleQueryHandler(peers);
```

Comment:
The comment at the top of the file says:
> // Sample query handler that will use all queryable peers within the network to evaluate transactions, with preference
> // given to peers within the same organization.

This code is only using the peers within the same organization. It should be passing the handler a `peers` array consisting of all the peers within the same organization followed by all the peers from other organizations. Something like:
```javascript
const orgPeers = network.channel.getEndorsers(network.mspid);
const otherPeers = network.channel.getEndorsers().filter((peer) => !orgPeers.includes(peer));
const allPeers = orgPeers.concat(otherPeers);
return new SampleQueryHandler(allPeers);
```

Reply:
fixed